content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -533,6 +549,14 @@ The global timeout of Accelerate has been exceeded. You can find the limit [here
 
 The user supplied invalid parameters. Currently only relevant for transaction methods. For example, setting a timeout that is too high. You can find the limit [here](/data-platform/accelerate/limitations#interactive-transactions-query-timeout-limit).
 
+#### <inlinecode>P6006</inlinecode> (<inlinecode>VersionNotSupported</inlinecode>)
+
+The chosen Prisma version is not compatible with Accelerate. This may occur when a user uses an unstable development version that we occasionally prune.
+
+#### <inlinecode>P6008</inlinecode> (<inlinecode>ConnectionError|EngineStartError</inlinecode>)
+
+Engine did not start. E.g. could not connect to database.
```

Comment:
What does the "E.g. could not connect to database." mean exactly? Maybe we can use a few more words to elaborate and provide context?